plugins/pixart-rf/fu-pxi-firmware.c

```diff
@@ -38,26 +40,33 @@ fu_pxi_rf_firmware_parse (FuFirmware *firmware,
 			     "firmware invalid, too small!");
 		return FALSE;
 	}
+	/* Get fw header */
```

Comment:
lowercase

Reply:
and it's a footer, no?

Reply:
Yes, we add the header to the end of fw bin

Reply:
a header at the end is called a footer.

Reply:
> > requested review from hughsie
> 
> Can you rebase against master and squash this down into one commit please.

Or just force merge?

Reply:
> Or just force merge?

Err, no. You need to rebase against master. Also please prefix your commits with `pxi: ` and use Sentence case. Thanks.